dist/tools/edbg/Makefile

```diff
@@ -4,16 +4,14 @@ PKG_VERSION=76f85abdea212ba23760723cce15e00ca4ae4b76
 PKG_LICENSE=BSD-3-Clause
 PKG_BUILDDIR=$(CURDIR)/bin
 
-# CC and CXX variables are set by RIOT's build process to cross compiling for
-# a specific target platfrom. Thus, we have to unset them to build edgb using
-# the system C/C++ compilers - i.e, to avoid conflicts on macOS systems
-CC=
-CXX=
+# Start in a clean environment, so variables set by RIOT's build process 
```

Comment:
does this comment still make sense here?

Reply:
Well, via `env …` we kinda start the edbg make in a clean environment, not?

Reply:
True. But the comment is far away from the ```env ...```. Without the context, I'd read the message as "[you have to] Start in a clean environment, so ...", and wonder what exactly I have to do...

Reply:
Ok. Will move the comment if possible.

Reply:
@kaspar030 Please have another look.